codacy/values-production.yaml

```diff
@@ -66,22 +66,22 @@ codacy-ingress:
   create: true
   ingress:
     hostname:
-      app: <--- codacy-app.dns.internal ---> # Codacy application DNS 
-      api: <--- codacy-api.dns.internal ---> # Codacy API DNS
+      app: <--- codacy-app.dns.internal ---> # Codacy application hostname
+      api: <--- codacy-api.dns.internal ---> # Codacy API hostname
     annotations:
       kubernetes.io/ingress.class: nginx
       kubernetes.io/tls-acme: "true"
       certmanager.k8s.io/acme-challenge-type: http01
       certmanager.k8s.io/cluster-issuer: letsencrypt
-      external-dns.alpha.kubernetes.io/hostname: '<--- codacy-app.dns.internal --->, <--- codacy-api.dns.internal --->' # Codacy application and API DNS 
+      external-dns.alpha.kubernetes.io/hostname: '<--- codacy-app.dns.internal --->, <--- codacy-api.dns.internal --->' # Codacy application and API hostnames
       external-dns.alpha.kubernetes.io/scope: public
       nginx.ingress.kubernetes.io/proxy-body-size: 128m
       nginx.ingress.kubernetes.io/ssl-redirect: "true"
       nginx.ingress.kubernetes.io/use-regex: "true"
     tls:
```

Comment:
We will not be using https (tls/ssl) on our self hosted installations. I guess this should also be dropped out for now? 
(this comment applies to all other values files as well :) )